common/arg_parser.h

```diff
@@ -141,13 +141,26 @@ class BitSet {
       bits_[i] = 0;
     }
   }
+  void fill(size_t bit = SIZE) {
+    for (size_t i = 0; i < NELEM(bits_); i++) {
```

Comment:
The loop should only iterate over all bits_[] where it can assign 0xFFFFFFFF, then assign the final value after the loop.